core/internal/cltest/cltest.go

```diff
@@ -1683,3 +1687,48 @@ func ClearDBTables(t *testing.T, db *sqlx.DB, tables ...string) {
 	err = tx.Commit()
 	require.NoError(t, err)
 }
+
+// CopyFlagSetFromAction takes the given flagSet and applies the actions flag to it.
+// `parentCommand` will filter the app commands and only applies the flags if the command/subcommand has a parent with that name, if left empty no filtering is done
+func CopyFlagSetFromAction(action interface{}, flagSet *flag.FlagSet, parentCommand string) {
```

Comment:
What do you think about using `Apply` instead of `Copy` in the name?
```suggestion
// FlagSetApplyFromAction applies the flags from action to the flagSet.
func FlagSetApplyFromAction(flagSet *flag.FlagSet, action interface{}, parentCommand string) {
```
It would be nice to have more type safety, but if we must use `interface{}` then let's document the expectations here.

Reply:
I agree `FlagSetApplyFromAction` is a better name.
Changed in [0b548f9](https://github.com/smartcontractkit/chainlink/pull/8173/commits/0b548f91b35e5fcb6c95002d26530e1faf342b43)